### ISSUE TITLE
Wishlist: beep instead of voice message for some announcements

### ISSUE THREAD

**Issue by RicoZ-OSM** [osmandapp/Osmand#1739]
For example when exceeding the speed limit a beep (and displaying the limit) might be preferable.. much quicker, does not conflict with other announcements etc. The spoken warning takes too long to speak and requires more driver attention to listen to.

A beep and a large enough symbol on the screen might also be preferable for pedestrian crossings and other warnings.

**Comment by Gizmot** [comment]
Asked the same thing before: beep instead of "you are going above the speed limit" annoyance and a flashing max speed limit icon

**Comment by Barrocas** [comment]
Maybe use a shorter warning like:
'Max speed xxx km/h'?
And a flashing signal?

**Comment by RicoZ-OSM** [comment]
we cold probably edit some configuration file to tweak the voice message to our liking. Hm.. maybe could we just tweak it to say "beeeep"?

I think there would be enough demand to make that easier configurable though.

There is also the issue that sometimes it would be very nice to have the warning in advance - when driving on a 100km/h road and getting into a village with a 50km/h limit the voice message might be completed only after you drive several hundreds of meters through the village.

**Comment by Barrocas** [comment]
Sorry but I disagree, 'beeeep' may sound as a warning, but it doesn't
inform of anything in particular... If it's going to speak up, it is better
to have some sence in it.

Some users travel with osmand with screen turned off, hearing only the
instructions, either to save battery, or like when riding motorcicle and
don't have a place to hold the phone, except pockets.

I'd rather have short separate announcements to replace (or add) the
generic 'attencion' warning, for things like 'bump', 'toll', maybe
'crossing', etc.

As for the speeding warning, maybe 'speed' or 'speed limit', or as stated
before, 'speed limit xxxkm/h' (I would prefer the last, since it's much
more complete)?
Any of these options would be shorter to TTS than the 'you are exceeding
the speed limit'.



-- 
Cumprimentos,
D.er Hugo Barrocas

**Comment by RicoZ-OSM** [comment]
it depends on situation what is better and it should be configurable. For users having the display in view  a beep with a flashing info screen would be optimal to warn about a condition needing attention. Btw even if you are saving battery power the screen could still go on to display warnings just as it goes on to display crossings and similar.

Even when hidden in a pocket it might be a good idea to start warning messages with a beep to distinguish them from informational messages.

**Comment by Barrocas** [comment]
Yes, I know the screen may have the 'turn on' setting, but when riding on
motorcicle that function doesn't have any use, if the phone is inside a
pocket in your jacket...

That's why I believe that a short TTS voice warning is better, serves all
users in a broader way.

I do use Osmand in the Car, with a bluetooth headset inside my ear, so
voice by TTS is fine, but I really don't think I would like a Beeep inside
my ear.
It's just my opinion.


-- 
Cumprimentos,
D.er Hugo Barrocas

**Comment by RicoZ-OSM** [comment]
In an effort to workaround the issue I have created a new language under "voice", copied and edited an existing ttsconfig.p to

`string('exceed_limit.ogg', 'biep biep ').`

This kind of works - however is not loud enough. How would I put a sound file into this place??

**Comment by polarbearing** [comment]
In #2496 I asked for the variables for speed and maxspeed to be provided for the TTS files. 
A short "Hey 50" is as long as a beep and provides more information to the driver without the need to take the view from the traffic.

**Comment by sonora** [comment]
- Shorter speed limit prompts will be implemented via #2496
- Some other messages have also been shortened in recent weeks, will be in next release.
- Further shorter wording may be found individually per language, so please create pull request individually. (Please compare your language to the  English implementation: We use "After x meters turn ..." for the first (preparation) prompt, the "In y meters turn..." for the next prompt. Some languages may still have obsolete wording like "Prepare for..." etc. which I think should be cleaned out. Also: Instead of "Follow the course of the road..." you may just take something like "Continue...")
- I did not see many votes for generic "beeps" in recent months, and think tis may in deed be a little "weird"
